src/components/Input.tsx

```diff
@@ -43,8 +44,11 @@ const Input = styled.input`
 
   :disabled {
     // Overrides WebKit's override of input:disabled color.
-    -webkit-text-fill-color: ${({ theme }) => transparentize(1 - loadingOpacity, theme.primary)};
-    color: ${({ theme }) => transparentize(1 - loadingOpacity, theme.primary)};
+    -webkit-text-fill-color: ${({ theme }) => inputValueColorTransparentize(theme.primary)};
```

Comment:
Can you use opacify directly in these instances, instead of creating a helper function in L6? I think it is small enough that inlining is more readable.

Also, prefer opacify if it exists over transparentize, to avoid needing add'l math.

Reply:
I'd prefer the changes to this file be backed out - I think it makes this harder to read.

Reply:
it exists, but it doesn't work quite the way you're thinking. 

transparentize is the tool for reduction, opacify for increases.